wpilibc/src/main/native/include/frc/event/BooleanEvent.h

```diff
@@ -130,6 +133,7 @@ class BooleanEvent {
 
  private:
   EventLoop* m_loop;
-  std::function<bool()> m_condition;
+  std::shared_ptr<std::function<bool()>> m_condition;
```

Comment:
What’s the reason to make the function a shared_ptr? Function is copyable.

Reply:
Left over from an old implementation.